website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -4,391 +4,275 @@ linkTitle: Releasing
-pool.sks-keyservers.net or any other public keyserver that synchronizes
-the key to other public keyservers. Once that is done, other people can download your public
-key and you are ready to sign releases.
-
-## Verifying a release
+### 6. Fast-forward the `latest` and release branches
 
-To verify a git tag, use this command:
+The `latest` branch points to the latest release tag to keep the main website subdomain up-to-date.
+Run the following commands to do so:
 
 ```sh
-$ git verify-tag --verbose "$TAG_NAME"
+git checkout latest
+git reset --hard tags/v1.3.0
+git push -f upstream latest
 ```
 
-If you do not have the mantainers public key on your machine, you will get an error message similiar to this:
-
-```console
-$ git verify-tag --verbose "$TAG_NAME"
-object 61e0c23e9d2e217f8d95ac104a8f2545c102b5c3
-type commit
-tag v0.6.0
-tagger Ish Shah <ishah@redhat.com> 1552688145 -0700
-
-Operator SDK v0.6.0
-gpg: Signature made Fri Mar 15 23:15:45 2019 CET
-gpg:                using RSA key <KEY_ID>
-gpg: Can't check signature: No public key
-```
-
-To download the key, use the following command, replacing `$KEY_ID` with the RSA key string provided in the output of the previous command:
+Similarly, to update the release branch, run:
 
 ```sh
-$ gpg --recv-key "$KEY_ID"
+git checkout v1.3.x
+git reset --hard tags/v1.3.0
+git push -f upstream v1.3.x
 ```
 
-To verify a release binary using the provided asc files see the [installation guide.][install-guide]
+### 7. Post release steps
 
-## Release steps
+See the [post-release section](#post-release-steps).
 
-These steps describe how to conduct a release of the SDK, upgrading from `v1.2.0` to `v1.3.0`.
-Replace these versions with the current and new version you are releasing, respectively.
 
-For major and minor releases, `master` should be locked between steps 3 and 6 so that all commits will be either in the new release
-or have a pre-release version, ex. `v1.2.0+git`. Otherwise commits might be built into a release that shouldn't be.
-For patch releases, ensure all required bugs are [cherry-picked](#cherry-picking), then the release branch `v1.3.x` should be locked down.
+## Patch releases
 
-### 1. Update OLM bindata
+We will use the `v1.3.1` release version in this example.
 
-This step is to be preferably performed only during major releases. Update the `OLM_VERSION` variable in Makefile to the latest successful release of OLM. Run `make bindata` so that `internal/bindata/olm` is updated. Also, update the `availableVersions` map in `internal/bindata/olm/versions.go` to contain the version of OLM which you have specified in the Makefile.
+### Before starting
 
-Submit a PR with the changes and merge it with master.
+Create and merge a commit that updates the top-level [Makefile] variable `IMAGE_VERSION`
+to the upcoming release tag `v1.3.1`. This variable ensures sample projects have been tagged
+correctly prior to the release commit.
```

Comment:
Same comment as above, can we add a block that says either a sed command to do it or literally says:

```sh
vi Makefile
# update IMAGE_VERSION
```
Otherwise, I will miss this.